.gitlab-ci.yml

```
@@ -116,3 +117,61 @@ checksums-sign:
   artifacts:
     paths:
       - signed/
+
+run-overhead-test:
+  stage: run-overhead-test
+  rules:
+    - if: '$CI_COMMIT_REF_NAME == "main" && $CI_COMMIT_MESSAGE =~ /\[agent-version-update\]/'
```

Will we have to introduce some commit message conventions for this to work?

---

yes

---

I am not sure, if the wording should contains agent. We are not using agent if it is related to .NET.

```suggestion
    - if: '$CI_COMMIT_REF_NAME == "main" && $CI_COMMIT_MESSAGE =~ /\[signalfx-version-update\]/'
```